gemfirexd/shared/src/main/resources/com/pivotal/gemfirexd/internal/loc/toolsmessages.properties

```diff
@@ -458,6 +458,30 @@ FS_ADDRESS_ARG=\
 \                        (default 80% of critical-off-heap-percentage)]\n\
 \                      [-host-data=<true|false> (default true)]\n\
 \                      [-log-file=<path> (default gfxdserver.log)]
+#snappy specific message, hides mcast port args, offheap-size renamed to memory-size etc.
+FS_SNAPPY_ADDRESS_ARG=\
+\                      [-sync=<false|true> (default false)]\n\
+\                      [-heap-size=<size>] [-memory-size=<size>]\n\
+\                      [-locators=<addresses>] [-start-locator=<address>]\n\
+\                      [-server-groups=<groups>] [-lock-memory]\n\
+\                      [-rebalance] [-init-scripts=<sql-files>]\n\
+\                      [-bind-address=<addr> (default is hostname or localhost\n\
+\                        if hostname points to a local loopback address)]\n\
+\                      [-run-netserver=<true|false> (default true)]\n\
+\                      [-client-bind-address=<clientaddr> (default is localhost)]\n\
+\                      [-client-port=<clientport> (default 1527)]\n\
+\                      [-critical-heap-percentage=<critical-heap-percentage>\n\
+\                        (default 90% if -heap-size is provided, otherwise not \n\
+\                         configured)]\n\
+\                      [-eviction-heap-percentage=<eviction-heap-percentage>\n\
+\                        (default 80% of critical-heap-percentage)]\n\
+\                      [-critical-off-heap-percentage=<critical-off-heap-percentage>\n\
+\                        (default 90% if -off-heap-size is provided, otherwise not\n\
+\                         configured)]\n\
+\                      [-eviction-off-heap-percentage=<eviction-off-heap-percentage>\n\
+\                        (default 80% of critical-off-heap-percentage)]\n\
+\                      [-host-data=<true|false> (default true)]\n\
+\                      [-log-file=<path> (default gfxdserver.log)]
```

Comment:
also remove the options mentioned from here (lock-memory, start-locator, *off-heap-percentate)